elixir/lib/contracts.ex

```diff
@@ -15,13 +15,20 @@ defmodule Trento.Contracts do
     id = Keyword.get(opts, :id, UUID.uuid4())
     source = Keyword.get(opts, :source, "trento")
     data = Protobuf.Encoder.encode(struct)
+    seconds = Keyword.get(opts, :seconds, :os.system_time(:second))
```

Comment:
Now, thinking it better, instead of using `seconds`, I would pass the whole timestamp. Second looks a bit confusing.

So the unique change would, that we create the default timestamp as: 
```Google.Protobuf.Timestamp.new(seconds: :os.system_time(:second), nanos: 0)```, and we pass it to `:ce_timestamp`.

And in the test, we create a completely new timestamp with a known second.

Reply:
shouldnt we use `DateTime.utc_now()` ?

Reply:
The `Google.Protobuf.Timestamp` only accepts seconds and nanoseconds as input:
https://github.com/elixir-protobuf/google-protos/blob/master/lib/google_protos/timestamp.pb.ex#L5

Reply:
`System.os_time(:second)`